clients/iscmove/iscmoveclient/iscmoveclienttest/setup.go

```diff
@@ -15,14 +17,26 @@ import (
 func NewSignerWithFunds(t *testing.T, seed []byte, index int) cryptolib.Signer {
 	seed[0] = seed[0] + byte(index)
 	kp := cryptolib.KeyPairFromSeed(cryptolib.Seed(seed))
-	err := iotaclient.RequestFundsFromFaucet(context.TODO(), kp.Address().AsIotaAddress(), iotaconn.LocalnetFaucetURL)
+	err := iotaclient.RequestFundsFromFaucet(context.TODO(), kp.Address().AsIotaAddress(), l1starter.Instance().FaucetURL())
 	require.NoError(t, err)
 	return kp
 }
 
-func NewLocalnetClient() *iscmoveclient.Client {
+func NewIscMoveWebSocket(ctx context.Context, log *logger.Logger) (*iscmoveclient.Client, error) {
```

Comment:
```suggestion
func NewISCMoveWebSocket(ctx context.Context, log *logger.Logger) (*iscmoveclient.Client, error) {
```

Also, since this function is already in `iscmoveclient` package, I would name it something like `NewWebSocketClient` to avoid stuttering.